src/Microsoft.DotNet.Interactive.ApiCompatibility.Tests/ApiCompatibilityTests.cs

```diff
@@ -69,6 +70,13 @@ public void csharp_api_is_not_changed()
         this.Assent(contract, _configuration);
     }
 
+    [FactSkipLinux]
+    public void csharpProject_api_is_not_changed()
+    {
+        var contract = ApiContract.GenerateContract<CSharpProjectKernel>();
+        this.Assent(contract, _configuration);
```

Comment:
We need serialization contract tests in place for the commands and events from day 1 but otherwise I think this project is too new to both full API surface contract tests. A lot of these types might better be made internal. Let's review those later.

Reply:
ok will take this part off

Reply:
done